src/App.vue

```diff
@@ -676,7 +676,11 @@ export default {
       }
 
       // verify that NR type matches entity type from application
-      if (nr.legalType !== this.getEntityType) {
+      if (nr.legalType === 'CCC' && this.getEntityType === CorpTypeCd.BC_CCC) {
+        // TEMPORARY FIX:
+        // at the moment, Legal API is passing CCC instead of CC
+        // remove this when no longer needed
+      } else if (nr.legalType !== this.getEntityType) {
```

Comment:
... Until we sort out how to fix this (ongoing discussion in RocketChat).

Reply:
Sounds like a plan.